tests/compiler/test_caps.py

```diff
@@ -20,11 +20,11 @@ def test_cap():
     assert ckt_lib.find("TEST_CAP")
     primitives = PrimitiveLibrary(ckt_lib, pdk_path).gen_primitive_collateral()
     all_primitive_names = set([i.name for i in primitives if isinstance(i, SubCircuit)])
-    assert all_primitive_names == {'CAP_2T_I1', 'CAP_2T', 'NMOS_4T'}
-    assert primitives.find('CAP_2T_I1').elements[0].parameters == {'VALUE': '3E-14', 'PARALLEL': '1', 'STACK': '1'}
-    assert primitives.find('CAP_2T').elements[0].parameters == {'VALUE': '6E-14', 'PARALLEL': '1', 'STACK': '1'}
+    assert all_primitive_names == {'NMOS_4T_17941375', 'CAP_2T_57809468', 'CAP_2T_8193393'}
+    assert primitives.find('CAP_2T_8193393').elements[0].parameters == {'VALUE': '3E-14', 'PARALLEL': '1', 'STACK': '1'}
+    assert primitives.find('CAP_2T_57809468').elements[0].parameters == {'VALUE': '6E-14', 'PARALLEL': '1', 'STACK': '1'}
```

Comment:
@kkunal1408 , a new method `primitives.startswith()` could make some of these tests easier.

Reply:
Fair point. These test will still need refactoring as the schema evolves. Another option could be to add an additional attribute to the schema for testing purposes. Anyway, something to consider in a future PR as revising tens of tests every time is hard :)